promql/printer.go

```diff
@@ -180,7 +180,7 @@ func (node *VectorSelector) String() string {
 	labelStrings := make([]string, 0, len(node.LabelMatchers)-1)
 	for _, matcher := range node.LabelMatchers {
 		// Only include the __name__ label if its no equality matching.
```

Comment:
Perhaps amend the comment with the new additional condition? (Something like: "... or it is a selection via `__name__=`.)

Reply:
Good point, fixed.